spk/emby/src/installer.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Package
+PACKAGE="emby"
+DNAME="Emby"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+TMP_DIR="${SYNOPKG_PKGDEST}/../../@tmp"
+INSTALL_LOG="${INSTALL_DIR}/var/install.log"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="${PACKAGE}"
+GROUP="users"
+
+SERVICETOOL="/usr/syno/bin/servicetool"
+FWPORTS="/var/packages/${PACKAGE}/scripts/${PACKAGE}.sc"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Install busybox stuff
+    ${INSTALL_DIR}/bin/busybox --install ${INSTALL_DIR}/bin
+
+    # Create user
+    adduser -h ${INSTALL_DIR}/var -g "${DNAME} User" -G ${GROUP} -s /bin/sh -S -D ${USER}
+
+    # Correct the files ownership
+    chown -R ${USER}:root ${SYNOPKG_PKGDEST}
+
+    # Add firewall config
+    ${SERVICETOOL} --install-configure-file --package ${FWPORTS} >> /dev/null
+
+    # Log
```

Comment:
I can't remember, but wasn't this supposed to be removed? Or did you want to keep it around for the beta?